parquet2hive_modules/parquet2hivelib.py

```diff
@@ -263,3 +299,96 @@ def _get_common_prefixes(bucket, prefix=''):
     result = client.list_objects(Bucket=bucket, Prefix=prefix, Delimiter='/')
     return [prefix.get('Prefix') for prefix in result.get('CommonPrefixes', [])]
 
+
+PARQUET_CONVERSIONS = {
+    0: 'boolean',
+    1: 'int',
+    2: 'bigint',
+    4: 'float',
+    5: 'double',
+    6: 'binary',
+}
+
+
+def parquet2sql(schema, table_name, location, partitions):
+    fields = [parquet2sql_column(*f) for f in parquet2sql_fields(schema[1:], schema[0].num_children)]
+    fields_decl = ", ".join(fields)
+
+    if partitions:
+        columns = ", ".join("`{}` string".format(p) for p in partitions)
+        partition_decl = " partitioned by ({})".format(columns)
+    else:
+        partition_decl = ""
+
+    return "drop table if exists {0}; create external table {0}({1}){2} stored as parquet location '\"'{3}'\"'; msck repair table {0};".format(table_name, fields_decl, partition_decl, location)
+
+
+def parquet2sql_fields(schema, children):
```

Comment:
The docs say that `schema` is a list of `SchemaElements`, which actually represents the tree but in depth-first traversal. Isn't this function assuming it's breadth first traversal? I.e. for
```
   p
  / \
 c1 c2
 /
c3
```
it assumes `[p, c1, c2, c3]`, but depth-first traversal would be  `[p, c1, c3, c2]`